### ISSUE TITLE
FilmOn Sign in

### ISSUE THREAD

**Issue by moell9** [chrippa/livestreamer#1075]
To watch content from FilmOn in 480p you need to login.
Where can I set my username and password ?

**Comment by tp0** [comment]
I don't think you can unless someone makes a plugin specifically for that site.
